specs/protocol/predeploys.md

```diff
@@ -215,7 +206,59 @@ and burn tokens, depending on if the user is depositing from L1 to L2 or withdra
 
 [Implementation](https://github.com/kroma-network/kroma/blob/main/packages/contracts/contracts/universal/KromaMintableERC721Factory.sol)
 
-Address: `0x4200000000000000000000000000000000000017`
+Address: `0x420000000000000000000000000000000000000C`
 
 The `KromaMintableERC721Factory` is responsible for creating ERC721 contracts on L2 that can be used for
 depositing native L1 NFTs into.
+
+## SequencerFeeVault
+
+[Implementation](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/L2/SequencerFeeVault.sol)
```

Comment:
Please change the link to the kroma one, `BaseFeeVault` also. We use our contract